src/archivist/filter/index.js

```diff
@@ -20,6 +20,18 @@ const { CiceroMarkTransformer } = ciceroMark;
 
 const ciceroMarkTransformer = new CiceroMarkTransformer();
 
+/**
+ * Filter content and convert it to markdown
+ *
+ * @param {Object} config - Filter configuration
+ * @param {string} config.content - Content to filter
+ * @param {string} config.mimeType - Mime type of the given content
+ * @param {Object} config.documentDeclaration - Document declaration used to describe document in Open Terms Archive
+ * @param {string} config.documentDeclaration.location - URL which held the content
+ * @param {string|Object|Array} config.documentDeclaration.contentSelectors - Content selectors for the parts of the content to be retained. Can be a Range Selector, a CSS selector or an array of a combination of both
+ * @param {string|Object|Array} config.documentDeclaration.noiseSelectors - Content selectors for the parts of the content to be removed. Can be a Range Selector, a CSS selector or an array of a combination of both
```

Comment:
> The second sentence is risky of documentation rot. A documentation link ( `@see`) would be safer.

Reply:
Sorry I don't get what you mean.
You mean `Can be a Range Selector, a CSS selector or an array of a combination of both` is subject to rot?
I'm not sure where this documentation exists already.
Do you suggest I use a `@see src/archivist/services/documentDeclaration.js` ?

Reply:
Ok done, 

Not sure that the JSDoc is correct though. Here is my attempt

`* @param {string} config.documentDeclaration - @see {@link ./src/archivist/services/documentDeclaration.js}`

Reply:
I also understand it this way :+1: Until we actually generate the JSdoc I don't think we can check it for real 😄